### PR TITLE
Upgrade SurrealDb.Net to v0.7.0

### PR DESCRIPTION
**Pull request by Odonno**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Please reference the issue this PR will close: #_[issue number]_

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

Please make sure you've completed the relevant tasks for this PR, out of the following list:

- [ ] Code compiles correctly
- [ ] Created/updated tests
- [ ] Unit tests passing
- [ ] End-to-end tests passing
- [ ] Extended the documentation
- [ ] Provided sample for the feature

### CODE REVIEW

File: .github/workflows/healthchecks_surrealdb_ci.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ jobs:
           surrealdb_username: root
           surrealdb_password: root
           surrealdb_auth: true
-          surrealdb_strict: 
+          surrealdb_strict:
```
Comment (adamsitnik) [inline .github/workflows/healthchecks_surrealdb_ci.yml:43]: do we need this space here?

**Review by adamsitnik — APPROVED** [review]
LGTM, thank you @Odonno !

File: .github/workflows/healthchecks_surrealdb_ci.yml (hunk 1 above)
Comment (Odonno) [reply .github/workflows/healthchecks_surrealdb_ci.yml:43]: If you wait a few minutes, we could simply remove this field in the next release of this GitHub action

Comment (adamsitnik) [reply .github/workflows/healthchecks_surrealdb_ci.yml:43]: There is no rush, I just wanted to check whether this was an accidental change or something done on purpose.

Comment (Odonno) [reply .github/workflows/healthchecks_surrealdb_ci.yml]: Done

**Pull request merged**